libminifi/src/properties/Properties.cpp

```diff
@@ -187,8 +158,8 @@ void fixValidatedProperty(const std::string& property_name,
 }  // namespace
 
 // Load Configure File
-// If the loaded property is time-period or data-size validated and it has no explicit units ms or B will be appended.
-// If the loaded property is integer validated and it has some explicit unit(time-period or data-size) it will be converted to ms/B and its unit cut off
+// If the loaded property is time-period or data-size validated, and it has no explicit units ms or B will be appended.
+// If the loaded property is integer validated, and it has some explicit unit(time-period or data-size) it will be converted to ms/B and its unit cut off
```

Comment:
If we are adding punctuation, we should add (and remove) some more:
```
// If the loaded property is time period or data size validated, and it has no explicit units, then ms or B will be appended.
// If the loaded property is integer validated, and it has some explicit unit (time period or data size), it will be converted to ms or B, and its unit is cut off.
```

Also, these two lines of comment should be moved before the `fixValidatedProperty()` function. (And the "Load Configure File" comment could be removed.)